docs/status.md

```diff
@@ -18,128 +18,128 @@ one of the following tracking labels.
 
  - Generic labels
     - **yes**: complete
-    - **wip**: semi-complete: Work in progress or under review.
     - **no**: not complete yet, but part of [the roadmap](https://github.com/openxla/stablehlo#roadmap).
- - Customized labels
-    - Verifier
-       - **match-xla**:  verifier in sync with  [XLA semantics](https://www.tensorflow.org/xla/operation_semantics).
-       - **match-spec**: verifier in sync with [StableHLO semantics](https://github.com/openxla/stablehlo/blob/main/docs/spec_draft.md).
-
+ - Customized labels for Verifier and Type Inference 
+    - **yes\***:  in sync with  [XLA semantics](https://www.tensorflow.org/xla/operation_semantics).
```

Comment:
Typo: double whitespace in two spots on this line.

Reply:
Done.